src/pages/Bridge/BridgeModal/BridgeModal.tsx

```diff
@@ -78,6 +78,11 @@ export const BridgeModal = ({
       setIsWarning(false)
       setBridgeName('Arbitrum One Bridge')
     }
+
+    if (activeBridge === 'xdai') {
+      setIsWarning(false)
+      setBridgeName('xDai Bridge')
```

Comment:
```suggestion
      setIsWarning(false)
      setBridgeName('xDai Bridge')
```

https://github.com/levelkdev/swapr-dapp/blob/1f199786ccc94fcdb508d7c5488c6ba5a540b33b/src/lib/eco-router/hooks.ts#L41

Suggest to batch the state updates like its done in hooks